membership-attribute-service/app/repositories/MembershipAttributesSerializer.scala

```diff
@@ -24,13 +24,13 @@ case class MembershipAttributesSerializer(tableName: String)
     Map(
       AttributeNames.userId -> membershipAttributes.UserId,
       AttributeNames.membershipNumber -> membershipAttributes.MembershipNumber.getOrElse(""),
-      AttributeNames.tier -> membershipAttributes.Tier
+      AttributeNames.tier -> membershipAttributes.Tier.getOrElse("")
     ).filter(_._2.nonEmpty).map(mkAttribute[String])
 
   override def fromAttributeMap(item: collection.mutable.Map[String, AttributeValue]) =
     Attributes(
       UserId = item(AttributeNames.userId),
       MembershipNumber = item.get(AttributeNames.membershipNumber).map(_.getS),
-      Tier = item(AttributeNames.tier)
+      Tier = Some(item(AttributeNames.tier))
```

Comment:
Tier is just wrapped in a Some, but how about if it was written as a ""?  It would be filtered out and not written, so would `item` have thrown an exception? or returned ""?  whatever actually happens, it should end up returning a None presumably so that it's symmetrical with toAttributeMap